### ISSUE TITLE
Not work on strapi v4.5.2

### ISSUE THREAD

**Issue by whydnxx** [ComfortablyCoding/strapi-plugin-transformer#50]
Not supported in new strapi versions.

 "@strapi/strapi": "4.5.2",
  "engines": {
    "node": ">=14.19.1 <=18.x.x",
    "npm": ">=6.0.0"
  },

**Comment by ComfortablyCoding** [comment]
Wierd that should have been fixed in 2.1.1. what error does it show?

**Comment by jacobedawson** [comment]
@ComfortablyCoding - I'm just learning of the horrors of GraphQL implementation using Strapi 4+, and can confirm that the transformer does not seem to be working for me on v4.5.3 - there are no errors, it just simply is not transforming responses, failing silently.

**Comment by ComfortablyCoding** [comment]
You are correct that this does not work for graphql. The reason being that it is not easily achievable to modify the generated gql schemas so their would be a mismatch.

Good point about the note, I will add it in.

**Comment by ComfortablyCoding** [comment]
Note that GraphQL is not supported has been added to the README

P.S. context #23 & #13
